packages/angular/src/generators/library/library.spec.ts

```diff
@@ -893,20 +894,20 @@ describe('lib', () => {
 
         expect(tsConfigLibJson2.exclude).toEqual([
           'src/**/*.spec.ts',
-          'src/test-setup.ts',
-          'jest.config.ts',
           'src/**/*.test.ts',
+          'jest.config.ts',
           'jest.config.cts',
```

Comment:
I know the change here is just ordering of the excludes, but isn't it odd to have both, if above we're checking specifically for .cts?